Sources/TestHelper/TestCase.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 import XCTest
-@testable import MapboxDirections
```

Comment:
testable not needed and also cause error in [MapboxNavigation.xcodeproj](https://github.com/mapbox/mapbox-navigation-ios/tree/main/MapboxNavigation.xcodeproj)
> ‘Module 'MapboxDirections' was not compiled for testing’